docs/business-apps/power-automate/guidance/manage-list-flows.md

```diff
@@ -39,7 +39,7 @@ For triggers that support run-only users, below the **Owners** card, a **Manage
 
 ![Manage run-only users card dropdown](../../../images/manage-run-only-users-card.png)
 
-Here, you can manage the run-only users or assign the users of the SharePoint list as run-only users. This means users who have read permissions to that list can run flows.
+Here, you can manage the run-only users or assign the users of the Microsoft list as run-only users. In order to run flows, users must have Edit Permissions on the list. Learn more about [list permissions and permission levels](https://docs.microsoft.com/sharepoint/understanding-permission-levels#list-permissions-and-permission-levels).
```

Comment:
Please convert the absolute URL to a relative link, per MSDocs red’s and listed in the build check report in the PR comments.

Reply:
How do you use a relative URL to refer to an article that is not in the dev docs, but another part of docts. microsoft.com?

Reply:
@andrewconnell I tried using a relative URL but it wasn't working. If there is a way to relatively refer to a doc outside of dev docs, please let me know.

Reply:
Just remove the FQDL like you see I've done. When in the same repo, you want to use the relative link to the file to get the automatic link checking by msdocs' automation checks, but in this case, just remove the protocol and domain.